tests/node-tests.js

```diff
@@ -31,7 +31,9 @@ function assertLongSampleParsedCorrectly(parsedCsv) {
 		"linebreak": "\n",
 		"aborted": false,
 		"truncated": false,
-		"cursor": 1209
+		"cursor": 1209,
+		"startByte": 0,
+		"endByte": 1209
```

Comment:
this is not success when run with node.js stream
this same assertion is working with node.js in sync mode but failed with node.js stream

Reply:
startByte for node.js stream case is 1301

Reply:
Should we add todo for this case ?. i think we can come back later. i had the same issue this afternoon trying to fix test cases.

Reply:
further inspection, this kind of make sense. We assert the completedResult when parsing. For sync case, it only run in single chunk so the meta is {startByte: 0, endByte: 1209}. For Stream use case, the file is read in 2 chunk and the meta is the data of second chunk with is {startByte: 1031 endByte: 1209}.

Reply:
@JohnLeSked I update the code to return startByte of the first chunk instead of the last chunk.